### PR TITLE
Actors to push metadata to AWS SNS

### PR DESCRIPTION
**Pull request by henriqueribeiro**
Reactivating old [PR](https://github.com/broadinstitute/cromwell/pull/5748) from @markjschreiber.

### CODE REVIEW

File: services/src/main/scala/cromwell/services/metadata/impl/aws/EventBridgeMetadataServiceActor.scala [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.
```
Comment (markjschreiber) [inline services/src/main/scala/cromwell/services/metadata/impl/aws/EventBridgeMetadataServiceActor.scala:2]: Is this actually AWS copyright?

**Review by markjschreiber — APPROVED** [review]

**Review by markjschreiber — COMMENTED** [review]
Apologies I probably wasn't too clear about the Copyright block. You can remove it from any classes that you developed. Any classes that are copied from the PR that I made to Cromwell should retain the copyright. Essentially, if I wrote the original then keep it, if you wrote it then remove or replace with your copyright.

**Comment by henriqueribeiro** [comment]
> Apologies I probably wasn't too clear about the Copyright block. You can remove it from any classes that you developed. Any classes that are copied from the PR that I made to Cromwell should retain the copyright. Essentially, if I wrote the original then keep it, if you wrote it then remove or replace with your copyright.

Got it. I will add it back and use yours on mine too

**Comment by henriqueribeiro** [comment]
@markjschreiber done

**Pull request merged**